ams/server/src/main/resources/mysql/update.sql

```diff
@@ -1,2 +1,42 @@
 -- If you have any changes to the AMS database, please record them in this file.
--- We will confirm the corresponding version of these upgrade scripts when releasing.
\ No newline at end of file
+-- We will confirm the corresponding version of these upgrade scripts when releasing.
+
+UPDATE table_optimizing_process
```

Comment:
Wouldn't it be better to handle it at the code level?

Reply:
Fixed. abandoning compatibility with historical tasks.